wezterm-font/src/locator/font_config.rs

```diff
@@ -65,6 +65,20 @@ impl FontLocator for FontConfigFontLocator {
             Ok(matches)
         }
 
+        fn to_handle(pat: FontPattern) -> anyhow::Result<FontDataHandle> {
+            let file = pat.get_file()?;
+            let index = pat.get_integer("index")? as u32;
+            let variation = index >> 16;
+            let index = index & 0xffff;
+            Ok(FontDataHandle {
+                source: FontDataSource::OnDisk(file.into()),
+                index,
+                variation,
+                origin: FontOrigin::FontConfig,
```

Comment:
Good idea, done! I called it `FontConfigMatch(…)` because we don't actually know it's an alias, we just use this resolution in specific circumstances.